datasets/NIHE-Northern-Ireland-homelessness-bulletin/main.py

```diff
@@ -0,0 +1,729 @@
+        
+        observations = quarter.fill(DOWN).is_not_blank() - footnote
+        
+        dimensions = [
+                    HDimConst('Measure Type','Households'),
+                    HDimConst('Unit','Count'),
+                    HDimConst('Assessment Decision', 'Accepted'),
+                    HDim(age_category, 'Age Bracket', DIRECTLY, LEFT),
+                    HDim(year, 'Period', DIRECTLY, ABOVE), 
+                    HDim(quarter,'Quarter', DIRECTLY,ABOVE),
+        ]  
+        tidy_sheet = ConversionSegment(tab, dimensions, observations)
+        table = tidy_sheet.topandas()
+        table['Period'] = table['Period'].str.replace('\.0', '')
+        table['Period'] = table['Period'].str.replace(r'\(.*\).*', '') + '/' + table['Quarter'].str.replace(r'\(.*\).*', '')
+        table.drop(['Quarter'], axis=1, inplace=True)
+        trace.store('combined_dataframe_acceptance', table) 
+
+df = trace.combine_and_trace(title, 'combined_dataframe_acceptance').fillna('')
+
+df = df.replace({'ONS Geography code': 
+{'Antrim and Newtownabbey' : 'N09000001', 
+'Ards and North Down' : 'N09000011',
+'Armagh City, Banbridge and Craigavon': 'N09000002', 
+'Belfast' : 'N09000003',
+'Causeway Coast and Glens': 'N09000004', 
+'Derry City and Strabane' : 'N09000005',
+'Fermanagh and Omagh': 'N09000006', 
+'Lisburn and Castlereagh': 'N09000007',
+'Mid and East Antrim': 'N09000008', 
+'Mid Ulster': 'N06000010', 
+'Newry, Mourne and Down' : 'N09000010',
+'Northern Ireland' : 'N07000001'
+}})
+    
+df.rename(columns={'OBS':'Value', 'DATAMARKER':'Marker'}, inplace=True)
+df = df.replace({'Value': {'' : '0'}})
+df['Value'] = df['Value'].astype(float).round().astype(int)
+df.loc[(df['Marker'] == '*'), 'Marker'] = 'Statistical disclosure'
+
+df = df.replace({
+'Reason for Homelessness': {'' : 'All'}, 
+'Accommodation not Reasonable breakdown' : {'' : 'All'},
+'Intimidation breakdown': {'' : 'All'}, 
+'Release from facilities breakdown' : {'' : 'All'},
+'Household Composition': {'' : 'All'}, 
+'Priority need Category' : {'' : 'All'},
+'Housing Assessment Outcome': {'' : 'All'}, 
+'Age Bracket': {'' : 'All'},
+'Assessment Decision': {'' : 'All'},
+})
+
+df = df[['Period', 'Reason for Homelessness', 'Accommodation not Reasonable breakdown', 'Intimidation breakdown', 'Release from facilities breakdown', 'Household Composition', 'ONS Geography code', 'Priority need Category', 'Housing Assessment Outcome', 'Age Bracket', 'Assessment Decision', 'Measure Type', 'Unit', 'Value', 'Marker']]
+
+cubes.add_cube(scraper, df, scraper.dataset.title)
+
+
+# -
+
+def with_date_overrides(period_dimension):
+    """
+    We're going to add a cellvalue overrides to each cell within the dimension AFTER
+    it's been defined.
+    So replacing the value of any dimensions cells that are blank with the appropriate year.
+    """
+    not_blank_cells = [x for x in period_dimension.hbagset if x.value != '']
```

Comment:
This is CLOSEST UP behaviour when doing the conversion_segment construction (i.e. `HDim(..., CLOSEST, UP)`)